tests/UI/pages/BO/orders/view.js

```diff
@@ -279,6 +291,15 @@ class Order extends BOBasePage {
     return this.elementVisible(page, this.returnProductsButton, 2000);
   }
 
+  /**
+   * Click on view invoice button to download the invoice
+   * @param page {Page} Browser tab
+   * @returns {Promise<void>}
+   */
+  async viewInvoice(page) {
```

Comment:
There is already a function that downloads documents, how this one is different ?

Reply:
The function downloadDocument used on documents tab, but this one is used on the heder of the page( Click on view invoice button).